api/pom.xml

```diff
@@ -29,5 +29,11 @@
 				<groupId>org.projectlombok</groupId>
 				<artifactId>lombok</artifactId>
 		    </dependency>
-	</dependencies>
+    <dependency>
+      <groupId>org.json</groupId>
```

Comment:
i dont think we need another Json parser Library , 
we already have `org.codehaus.jackson` that comes with the Core